templates/web/northumberland/about/faq-en-gb.html

```diff
@@ -0,0 +1,271 @@
+[% INCLUDE 'header.html', title = loc('Frequently Asked Questions'), bodyclass = 'twothirdswidthpage' %]
+
+[% INCLUDE 'about/_sidebar.html' %]
+
+<h1>Frequently Asked Questions</h1>
+
+<dl>
+
+<dt>What sort of issues can I report?</dt>
+<dd>
+<p>This site is for reporting things which are <strong>broken, dirty,
+damaged</strong> or <strong>dumped</strong>, and need <strong>fixing,
+cleaning</strong> or <strong>clearing</strong>.
+</dd>
+
+<dt>Which issues shouldn't I report?</dt>
+
+<dd>
+<p>Please don’t use this site to inform us of:</p>
+
+<ul>
+    <li>Urgent and emergency problems</li>
+    <li>Complaints about people, including anti-social behaviour</li>
+    <li>Issues with council services, such as bins and recycling</li>
+    <li>Proposals for change, eg to road layouts</li>
+    <li>Complaints about [% c.cobrand.council_name %]</li>
+</ul>
+
+<p>
+    For these types of issue, please
+    [%~ IF c.cobrand.feature('contact_us_url') %]
+     <a href="[% c.cobrand.feature('contact_us_url') %]">contact us</a>
+    [%~ ELSE %]
+    contact us
+    [%~ END %]
+    through the appropriate channel.
+</p>
+
+<b>Contact us by phone</b>
+
+<p>You can call us on 0345 600 6400:</p>
+
+<ul>
+    <li>Monday to Thursday 08:30 - 17:00</li>
+    <li>Friday 08:30 - 16:30</li>
+</ul>
+
+<p><b>PLEASE NOTE</b>: Calls outside of the hours above are for <b>emergency cases only</b> and will be answered by Northumberland Fire & Rescue Service.</p>
+
+<b>Customers with speed or hearing difficulties</b>
+
+<p>If you have speech of hearing difficulties you acn use Relay UK to contact us by dialing 018001 01670 623515. <a href="https://www.relayuk.bt.com/">Find more information about Relay UK</a>.</p>
```

Comment:
Couple of typos here:
```suggestion
<p>If you have speech or hearing difficulties you can use Relay UK to contact us by dialling 018001 01670 623515. <a href="https://www.relayuk.bt.com/">Find more information about Relay UK</a>.</p>
```